deploy/examples/create-external-cluster-resources.py

```diff
@@ -826,7 +938,37 @@ def validate_rados_namespace(self):
             raise ExecutionFailureException(
                 ("The provided rados Namespace, '{}', is not found in the pool '{}'").format(
                     rados_namespace, rbd_pool_name))
-
+    
+    def get_rgw_fsid(self):
+        access_key = self.out_map['RGW_ADMIN_OPS_USER_ACCESS_KEY']
+        secret_key = self.out_map['RGW_ADMIN_OPS_USER_SECRET_KEY']
+        rgw_endpoint = self._arg_parser.rgw_endpoint
+        base_url = "http://"
+        if self._arg_parser.rgw_tls_cert_path:
+            base_url = "https://"
+        base_url = base_url + rgw_endpoint + "/admin/info?"   
+        params = {'format': 'json'}
+        request_url = base_url + urllib.parse.urlencode(params)
+            
+        try:
+            r = requests.get(request_url, auth=S3Auth(access_key, secret_key,rgw_endpoint))
+        except requests.exceptions.Timeout:
+            raise ExecutionFailureException(
+                    "invalid endpoint:, not able to call admin-ops api{}".format(rgw_endpoint))
+        r1 = r.json()
+        if r1 is None or r1.get('info') is None:
+            return "NULL"
+        
+        return r1['info']['storage_backends'][0]['cluster_id']
+            
+    def validate_rgw_endpoint(self):
+        fsid = self.get_fsid()
+        rgw_fsid = self.get_rgw_fsid()
+        if fsid != rgw_fsid:
+            raise ExecutionFailureException(
+                ("The provided rgw Endpoint, '{}', is invalid. We are validating by calling the adminops api through rgw-endpoint and checking the cluster_id '{}' we get is same as the fsid of the ceph cluster '{}'").format(
```

Comment:
`checking the cluster_id '{}' we get is same as the` I don't think this is grammatically correct.